apps/site/assets/js/algolia-result.js

```diff
@@ -30,17 +30,23 @@ export const TEMPLATES = {
       <i aria-hidden="true" id="search-result__loading-indicator" class="fa fa-cog fa-spin c-search-result__loading-indicator"></i>
     </a>
   `),
-  projects: hogan.compile(`
+  threePlusIcons: hogan.compile(`
     {{#hasDate}}
     <div class="c-search-result__hit--vertical">
     {{/hasDate}}
-    <span>{{{hitIcon}}}</span>
     {{#id}}
     <a id="hit-{{id}}" class="${SELECTORS.result} c-search-result__link" href="{{hitUrl}}">
     {{/id}}
     {{^id}}
     <a class="${SELECTORS.result} c-search-result__link u-no-underline" href="{{hitUrl}}" data-queryid="{{analyticsData.queryID}}" data-hit-position="{{analyticsData.position}}" data-objectid="{{analyticsData.objectID}}">
     {{/id}}
+      <span>{{{hitIcon}}}</span>
+      <span class="c-search-result__feature-icons">
+      {{#hitFeatureIcons}}
+        {{{.}}}
+      {{/hitFeatureIcons}}
+      <br/>
+    </span>
```

Comment:
Would it work to put these lines immediately before the <a> on line 41? I'm thinking that taking them outside the link might help with the underlining issue.

Otherwise I think it makes sense to file a new ticket for that formatting and let that part go.